service/src/main/java/bio/terra/workspace/service/workspace/WorkspaceService.java

```diff
@@ -232,36 +275,23 @@ private UUID createWorkspaceWorker(
       @Nullable UUID sourceWorkspaceUuid,
       CloningInstructions cloningInstructions,
       @Nullable String projectOwnerGroupId,
+      @Nullable SpendProfile spendProfile,
+      @Nullable CloudPlatform cloudPlatform,
       AuthenticatedUserRequest userRequest) {
 
-    String workspaceUuid = workspace.getWorkspaceId().toString();
+    UUID workspaceUuid = workspace.getWorkspaceId();
     JobBuilder createJob =
-        jobService
-            .newJob()
-            .description(
-                String.format(
-                    "Create workspace: name: '%s' id: '%s'  ",
-                    workspace.getDisplayName().orElse(""), workspaceUuid))
-            .flightClass(WorkspaceCreateFlight.class)
-            .request(workspace)
-            .userRequest(userRequest)
-            .workspaceId(workspaceUuid)
-            .operationType(OperationType.CREATE)
-            .addParameter(
-                WorkspaceFlightMapKeys.WORKSPACE_STAGE, workspace.getWorkspaceStage().name())
-            .addParameter(WorkspaceFlightMapKeys.POLICIES, policies)
-            .addParameter(WorkspaceFlightMapKeys.APPLICATION_IDS, applications)
-            .addParameter(
-                WorkspaceFlightMapKeys.ResourceKeys.CLONING_INSTRUCTIONS, cloningInstructions)
-            .addParameter(SOURCE_WORKSPACE_ID, sourceWorkspaceUuid)
-            .addParameter(WorkspaceFlightMapKeys.PROJECT_OWNER_GROUP_ID, projectOwnerGroupId);
-
-    workspace
```

Comment:
SPEND_PROFILE_ID is no longer referenced anywhere (we pass the actual spend profile if we have it).